doc/source/handling-dependencies.rst

```diff
@@ -58,6 +58,12 @@ Runtime Environments
 
 A **runtime environment** describes the dependencies your Ray application needs to run, including :ref:`files, packages, environment variables, and more <runtime-environments-api-ref>`.  It is installed dynamically on the cluster at runtime.
 
+.. note::
+
+    This is an emerging feature.
```

Comment:
This is the opposite of GA right? Do we have an expected GA release that we can tell users?

Reply:
I think we should test all nightly tests with runtime env if we'd like to make it as a standard way to handle dependencies. Right now, the status quo is all of core nightly tests are running without the runtime env, which can easily cause reliability gap between us vs users.

Reply:
This is a good question.  I don't think there's a contradiction between a feature being GA while also not being as stable as Ray Core but I could be wrong here.  I agree with Sang that we should somehow signal to the user that runtime env is not as mature as the older approach.  Perhaps there's a better choice of words for this note.  @edoakes what are your thoughts on this?

Reply:
I think we can just say runtime environment is GA from version X.Y.Z here. I think the only thing we need to avoid is to advertise this feature as the "only way to handle dependencies", which the previous doc implied.